src/entities/volume.ts

```diff
@@ -0,0 +1,48 @@
+import { BigInt } from "@graphprotocol/graph-ts";
+import { NftContractDayData, NftContractHourData } from "../../generated/schema";
+import { BIG_INT_ZERO } from "../constants";
+
+function upsertContractDayVolumeData(
+    nftContractId: string,
+    value: BigInt,
+    timestamp: BigInt
+): void {
+    const dateEpochTime = timestamp.toI32() / 86400 * 86400; 
+    const dayDataId = `${nftContractId}-${dateEpochTime.toString()}`;
```

Comment:
Based on (Joe exchange subgraph)[https://github.com/traderjoe-xyz/joe-subgraphs/blob/main/subgraphs/exchange/src/exchange/enitites/token-day-data.ts#L14] - is there a need to do `BigInt.fromI32(dateEpochTime).toString()`?